SECURITY-INSIGHTS.yaml

```diff
@@ -1,11 +1,11 @@
 header:
   schema-version: 1.0.0
   expiration-date: '2024-09-28T01:00:00.000Z'
-  last-updated: '2025-12-12'
-  last-reviewed: '2025-12-12'
-  commit-hash: "84809c4a9f7ed5d83fdd7837f8ca0642881df846"
+  last-updated: ''
+  last-reviewed: ''
+  commit-hash: ""
```

Comment:
Oh, these seem empty.
Michal is OOO. So, I will open a follow-up PR.
cc @PBundyra